Pages/Cart/index.html

```diff
@@ -0,0 +1,294 @@
+                            <span class="card-txt-secondary">(40% OFF)</span>
+                        </h3>
+                    </div>
+                </div>
+
+            </div>
+            <div class="btn-container">
+                <div class="btn-standard-container">
+                    <button class="btn-rect btn-out-primary">Add to Cart</button>
+                    <button class="btn-icon btn-icon-secondary"><i class="fas fa-shopping-cart"></i></button>
+                    <button class="btn-icon btn-icon-secondary"><i class="far fa-heart"></i></button>
+                </div>
+            </div>
+        </div>
+
+        <!-- product 2 -->
+        <div class="card-basic ">
+            <div class="card-txt-container">
+                <img alt="card-img-product" class="basic-img" src="/assets/dog-bed.png">
+                <div class="txt-margin flex-row-between">
+                    <div>
+                        <h2 class="card-heading">Dog Bed</h2>
+                        <h3 class="card-subheading">&#8377 9,600 <span class="card-txt-red">&#8377 15,999 </span>
+                            <span class="card-txt-secondary">(40% OFF)</span>
+                        </h3>
+                    </div>
+                </div>
+
+            </div>
+            <div class="btn-container">
+                <div class="btn-standard-container">
+                    <button class="btn-rect btn-out-primary">Add to Cart</button>
+                    <button class="btn-icon btn-icon-secondary"><i class="fas fa-shopping-cart"></i></button>
+                    <button class="btn-icon btn-icon-secondary"><i class="far fa-heart"></i></button>
+                </div>
+            </div>
+        </div>
+
+
+        <!-- product 3 -->
+        <div class="card-basic ">
+            <div class="card-txt-container">
+                <img alt="card-img-product" class="basic-img" src="/assets/dog-bed.png">
+                <div class="txt-margin flex-row-between">
+                    <div>
+                        <h2 class="card-heading">Dog Bed</h2>
+                        <h3 class="card-subheading">&#8377 9,600 <span class="card-txt-red">&#8377 15,999 </span>
+                            <span class="card-txt-secondary">(40% OFF)</span>
+                        </h3>
+                    </div>
+                </div>
+
+            </div>
+            <div class="btn-container">
+                <div class="btn-standard-container">
+                    <button class="btn-rect btn-out-primary">Add to Cart</button>
+                    <button class="btn-icon btn-icon-secondary"><i class="fas fa-shopping-cart"></i></button>
+                    <button class="btn-icon btn-icon-secondary"><i class="far fa-heart"></i></button>
+                </div>
+            </div>
+        </div>
+
+
+
+    </div>
+    <!-- Footer -->
+    <footer>
+        <h4>Created by Vanshita Moolchandani</h4>
```

Comment:
![review2](https://user-images.githubusercontent.com/68228131/154761596-50307bd1-2bac-40b3-b0a2-8d039603ad96.png)

you can remove those blue lines by using text-decoration: "none" on links.